THIRD_PARTY.txt

```diff
@@ -11,7 +11,7 @@
 ###########
 
 This program is free software: you can redistribute it and/or modify
-it under the terms of the GNU General Public License verions 2,
+it under the terms of the GNU General Public License version 2,
```

Comment:
License files can be incredibly picky about changes, but even so, I'd hope this is a reasonable change.